sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/models/MetricsClientAudience.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.monitor.query.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * The audience indicating the authorization scope of metrics clients.
+ */
+public class MetricsClientAudience extends ExpandableStringEnum<MetricsClientAudience> {
```

Comment:
This should be named `MetricsAudience` to be consistent with the names used in other libraries.